backend/plugin/db/clickhouse/query.go

```diff
@@ -51,9 +51,7 @@ func (driver *Driver) QueryConn(ctx context.Context, conn *sql.Conn, statement s
 		queryResult, err := func() (*v1pb.QueryResult, error) {
 			rows, err := conn.QueryContext(ctx, statement)
 			if err != nil {
-				// ClickHouse will return "driver: bad connection" if we use non-SELECT statement for Query(). We need to ignore the error.
-				// nolint
-				return nil, nil
+				return nil, err
```

Comment:
This makes panic to err which isn't ideal either. Maybe we can employ something like below.

https://github.com/bytebase/bytebase/blob/d5d60635a951e6e7ac971e2d43e6ba22f91f80f9/backend/plugin/db/pg/pg.go#L664

Reply:
Thank you for the feedback! I'll look into how we can handle this more gracefully based on your suggestion.